detect_secrets/plugins/base.py

```diff
@@ -169,18 +169,18 @@ def build_assignment_regex(
         opt_quote = r'(?:"|\'|)'
         opt_open_square_bracket = r'(?:\[|)'
         opt_close_square_bracket = r'(?:\]|)'
-        opt_dash_undrscr = r'(?:_|-|)'
+        opt_dash_underscore = r'(?:_|-|)'
```

Comment:
Happy to drop (this or anything else). Notably, in principle, this should really be `opt_underscore_dash`, but...

Note: Google Sheets made this suggestion (and it surprised me). I wouldn't have.

Reply:
I'm okay with this.